pkg/build/nodeimage/const_storage.go

```diff
@@ -87,7 +87,7 @@ spec:
         app: local-path-provisioner
     spec:
       nodeSelector:
-        node-role.kubernetes.io/master: ''
```

Comment:
this label is deprecated, however we do want to only run on linux nodes for our friends doing windows join hijinks.